README.md

```diff
@@ -1,20 +1,33 @@
 video-mode-plugin
 =================
 
-A BrightAuthor plugin that changes the video resolution, frame rate, color space and color depth.
+A BrightSign plugin that changes the video resolution, frame rate, color space and color depth.
 
-# Usage
+SetSingleScreenVideoMode.brs is used for series 4 and below.
```

Comment:
md file does not automatically add new line break. You need to use "\<br/>\" to indicate for a new line.
You can see the preview with the "Display the rich diff" button on the top right.